### ISSUE TITLE
Dataframe load data from excel and query using sql

### ISSUE THREAD

**Issue by Kumar3010** [dotnet/corefxlab#2995]
can we load excel data using this libraray and query the dataframe as we do in padnas using sql command
df.select("select col1,col2,from df")

**Comment by pgovind** [comment]
No, we can't do this yet. However, if you find a library that lets you read excel files and exposes it as an `IDataReader`, we should be able to read it into a DataFrame once https://github.com/dotnet/corefxlab/pull/2924/files goes in

**Comment by YohDeadfall** [comment]
There's [ExcelDataReader](https://github.com/ExcelDataReader/ExcelDataReader) which is ADO.NET compatible.

**Comment by Kumar3010** [comment]
@pgovind - could you please let us know if this is possilbe now

**Comment by pgovind** [comment]
Not yet unfortunately. #2924 is inactive at the moment. Would you be interested in adding unit tests to that PR? If not, this is on the list for the next DataFrame release. I'm tied up with .NET 6 work at the moment though, so it'll be a while before I can get to this.
